### DISCUSSION TITLE
Cannot open / view PDF in mobile browsers

### DISCUSSION THREAD

**Discussion by sunidy** [mozilla/pdf.js#18225] [Q&A]
The current stable version 4.3.136 cannot open PDF in mobile brows.

I tested it under iOS and Android with Safari and Chrome. 

Is there an issue or some settings need to be changed?

=======================
Feb 13, 2025
=======================
Just tested the latest version 4.10.38, the issue still not been fixed. 

I tested Chrome and Safari on iOS.

==========
March 19, 2026

Sorry for a late respond. We have found it caused by the browser in my old Android. :(

**Comment by sunidy** [comment]


**Reply by machmalpause** [reply]
Yes, tested and confirmed on several devices running iOS 16.7.8

**Comment by crli** [comment]
Hope to fix versions 4.267 and above

**Comment by roldeguz** [comment]
Any updates on this?

**Comment by sunidy** [comment]


**Reply by nicolo-ribaudo** [reply]
That first link works in Firefox and Chrome (on Android — because Chrome on iOS is actually Safari).

This seems like a Safari bug.

**Reply by sunidy** [reply]
But the second link works in Chrome and Safari on iOS.

Not sure what changed in PDF.js since version 4.9.379 that caused the problem.

**Reply by nicolo-ribaudo** [reply]
What version of Safari for iOS are you using? I just tested on my iPad and I can open both. It's possible that you might need to use the legacy build (https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#legacy-build), which is transpiled to support browsers other than the latest Chrome & Firefox versions.
